addons/web_editor/static/src/xml/editor.xml

```diff
@@ -19,22 +19,22 @@
                     <li id="pre-dropdown-item">
                         <a class="dropdown-item" href="#" id="pre" data-call="setTag" data-arg1="pre"><pre>Code</pre></a>
                     </li>
-                    <li id="heading1-dropdown-item">
+                    <li t-if="props.showHeading1" id="heading1-dropdown-item">
```

Comment:
Was there some prior options to do that ? Why would we want to show like heading 4 and 2 without the others ?

Reply:
This sounds like overengineering to me. We have like 5 or 6 different options just to enable or not some heading sizes... It feels very overkill to me. Not what you did, but the fact that we want this functionally while it has an impact in the code that is not worth the functional gain IMO. If we don't want to challenge the functional choice, couldn't we at least have some kind of compound option like `headings: [4, 5, 6]` for example ?

Reply:
`headings: [4, 5, 6]` is not clear to me.
It does not tell what it restricts the visibility of the headings.

Also, all options the format `show<button>` and at the exception of heading, the option would change. For which reason?
That format could imply that there is something different with that option when there is actually nothing different.